e2e/__snapshots__/templates.test.js.snap

```diff
@@ -7298,17 +7312,18 @@ exports[`Templates React InstantSearch widget File content: src/lib/widget.tsx 1
 "import { DatePickerComponent } from './component';
 import { connectDatePicker } from './connector';
 
-import type { ElementType } from 'react';
-
 type WidgetParams = {
   /**
    * Placeholder text for input element.
    */
   placeholder?: string;
 };
 
-export const DatePicker: ElementType<WidgetParams> =
-  connectDatePicker(DatePickerComponent);"
+export const DatePicker: React.ElementType<WidgetParams> = connectDatePicker(
+  DatePickerComponent,
+  // @ts-expect-error
```

Comment:
why does this expect an error? Should we make a PR to DefinitelyTyped before this?

Reply:
PR for new types here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60431